lab7.md

```diff
@@ -100,10 +100,11 @@ Check how many times the function has been called:
 ```
 $ faas list
 Function    Invocations
-issue-bot   1
+issue-bot   2
 ```
 
 Each time you create an issue the count will increase due to GitHub's API invoking the function.
+> _Github will invoke the function via a ping request once while setting up the webhook. That's why there are at least 2 invocations at this point_
```

Comment:
It was 1 for me when I tested it with Stefan, but perhaps we should change it to at least `1`?